devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -31,7 +31,7 @@ metadata {
 		status "no motion (basic)"  : "command: 2001, payload: 00"
 		status "motion (binary)"    : "command: 3003, payload: FF"
 		status "no motion (binary)" : "command: 3003, payload: 00"
-		status "wakeup" 			: "command: 8407, payload: 00"
+		status "wakeup" 			: "command: 8407, payload: "
```

Comment:
Just thought I'd point out that this was never showing up for me when I tested on the simulator so I would do a raw command of `8407`, but that was giving me outofboundsexceptions so I went with `840700` and that seemed to do what I wanted.